pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+###############################################################################
+[tool.poetry]
+name = "smqtk_relevancy"
+version = "0.15.0"
+description = "SMQTK Relevancy"
+authors = ["Kitware, Inc. <smqtk-developers@kitware.com>"]
+license = "BSD-3-Clause"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+numpy = "^1.19.5"
+pillow = "^8.2.0"
+smqtk-core = ">=0.18.0"
+smqtk-dataprovider = ">=0.15.0"
```

Comment:
Maybe for update later, but these should all be at >=0.16.0 now.

Reply:
updated